pilot/lib/pilot/build_manager.rb

```diff
@@ -10,11 +10,12 @@
 module Pilot
   class BuildManager < Manager
     def upload(options)
-      start(options)
+      start(options) unless options[:skip_waiting_for_build_processing] && options[:apple_id]
+      @config = options if options[:skip_waiting_for_build_processing] && options[:apple_id] # ugly as hell, we want to do that differently
```

Comment:
This is ugly as hell, but the only way to have `@config` be filled as it would be in `start`. I want to avoid calling `start()`, as it triggers the spaceship login.

Reply:
Actually, thinking about it, doing this here manually might _break_ further calls to `start` because it will prevent the spaceship login to happen later if the presence of `@config` is used as an indicator if a login has happened before or not.

Not a problem right now as we don't have any further calls of `start` or anything that then needs the spaceship login in this case, it is still ugly.

What would a better way to acheive the same thing be? Add a `spaceship_login_happened` bool to use instead of abusing `@config`?